utils/utils.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package google
+package utils
```

Comment:
could this go in `ancestrymanager` instead of `utils`?

Reply:
Done